openshift/e2e-common.sh

```diff
@@ -334,6 +337,7 @@ function run_e2e_tests(){
     --customdomain=$subdomain \
     --https \
     --resolvabledomain || failed=1
+  oc -n ${SYSTEM_NAMESPACE} patch knativeserving/knative-serving --type=merge --patch='{"spec": {"config": { "features": {"kubernetes.podspec-volumes-emptydir": "disabled"}}}}' || fail_test
 
   oc -n ${SYSTEM_NAMESPACE} patch knativeserving/knative-serving --type=merge --patch='{"spec": {"config": { "features": {"tag-header-based-routing": "enabled"}}}}' || fail_test
```

Comment:
Can we use the toggle function from `source "$(dirname "$0")/../test/e2e-common.sh"` or does it cause problems?

Reply:
No, unfortunately the toggle patches to configmap not `kknativeserving` object. So we cannot use it.